quotas_sample.txt

```diff
@@ -0,0 +1,6 @@
+developers /filestore/departments/developers /mnt/departments/developers 30.0
```

Comment:
@carlilek this is specific to your installation, can you remove this from the PR please?